lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
@markusweimer @tcNickolas based on this thread and the non obvious nature of extrapolating info from the java build  I'm inclined to keep these Prerequisites here